test/core/xds/xds_transport_fake.h

```diff
@@ -235,6 +245,7 @@ class FakeXdsTransportFactory : public XdsTransportFactory {
         ABSL_GUARDED_BY(&mu_);
     std::map<std::string /*method*/, RefCountedPtr<FakeStreamingCall>>
         active_calls_ ABSL_GUARDED_BY(&mu_);
+    std::function<void()> too_many_pending_reads_callback_;
```

Comment:
I don't think we need to store a copy of the callback in the `FakeXdsTransport` instance.  `FakeXdsTransport` already holds a ref to `FakeXdsTransportFactory`, so it can instead just access `factory_->too_many_pending_reads_callback_`.

Reply:
Done